[2025-05-04T16:19:50.828+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-04T16:19:50.875+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: google_maps_reviews.remove_duplicates manual__2025-05-04T16:19:25.800169+00:00 [queued]>
[2025-05-04T16:19:50.900+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: google_maps_reviews.remove_duplicates manual__2025-05-04T16:19:25.800169+00:00 [queued]>
[2025-05-04T16:19:50.903+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-05-04T16:19:50.931+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): remove_duplicates> on 2025-05-04 16:19:25.800169+00:00
[2025-05-04T16:19:50.956+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'google_maps_reviews', 'remove_duplicates', 'manual__2025-05-04T16:19:25.800169+00:00', '--job-id', '444', '--raw', '--subdir', 'DAGS_FOLDER/***dag.py', '--cfg-path', '/tmp/tmp05v883rn']
[2025-05-04T16:19:50.954+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=846) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-05-04T16:19:50.959+0000] {standard_task_runner.py:105} INFO - Job 444: Subtask remove_duplicates
[2025-05-04T16:19:50.960+0000] {standard_task_runner.py:72} INFO - Started process 868 to run task
[2025-05-04T16:19:51.056+0000] {task_command.py:467} INFO - Running <TaskInstance: google_maps_reviews.remove_duplicates manual__2025-05-04T16:19:25.800169+00:00 [running]> on host 8280c28b7f0f
[2025-05-04T16:19:51.247+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='google_maps_reviews' AIRFLOW_CTX_TASK_ID='remove_duplicates' AIRFLOW_CTX_EXECUTION_DATE='2025-05-04T16:19:25.800169+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-04T16:19:25.800169+00:00'
[2025-05-04T16:19:51.251+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-05-04T16:19:51.252+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-05-04T16:19:51.254+0000] {logging_mixin.py:190} INFO - Current task name:remove_duplicates state:running start_date:2025-05-04 16:19:50.877575+00:00
[2025-05-04T16:19:51.254+0000] {logging_mixin.py:190} INFO - Dag name:google_maps_reviews and current dag run status:running
[2025-05-04T16:19:51.256+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-05-04T16:19:51.274+0000] {base.py:84} INFO - Retrieving connection 'postgres_defaut'
[2025-05-04T16:19:51.288+0000] {logging_mixin.py:190} INFO - Duplicate reviews removed.
[2025-05-04T16:19:51.290+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-05-04T16:19:51.311+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-05-04T16:19:51.313+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=google_maps_reviews, task_id=remove_duplicates, run_id=manual__2025-05-04T16:19:25.800169+00:00, execution_date=20250504T161925, start_date=20250504T161950, end_date=20250504T161951
[2025-05-04T16:19:51.359+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-05-04T16:19:51.360+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-05-04T16:19:51.361+0000] {logging_mixin.py:190} INFO - Dag name:google_maps_reviews queued_at:2025-05-04 16:19:25.872407+00:00
[2025-05-04T16:19:51.362+0000] {logging_mixin.py:190} INFO - Task hostname:8280c28b7f0f operator:PythonOperator
[2025-05-04T16:19:51.383+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-04T16:19:51.446+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-05-04T16:19:51.451+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
